addons/lookatrandom/addon.json

```
@@ -0,0 +1,13 @@
+{
+  "name": "Point towards random direction",
+  "description": "Adds a \"random position\" option to the \"point towards ()\" block that makes the sprite point in a random direction. This will work for everyone using the project.",
+  "userscripts": [
+    {
+      "url": "code.js",
+      "matches": ["https://scratch.mit.edu/*"]
```

```suggestion
      "matches": ["projects"]
```
I _think_ I got that right.